custom/icds_reports/reports/bihar_api.py

```diff
@@ -68,3 +69,41 @@ def get_api_demographics_data(month, state_id, last_person_case_id):
     # To apply pagination on database query with data size length
     limited_demographics_data = list(demographics_data_query[:CAS_API_PAGE_SIZE])
     return limited_demographics_data,  get_total_records_count(month, state_id)
+
+
+@icds_quickcache(['month', 'state_id'], timeout=60 * 60 * 2)
+def get_total_school_records_count(month, state_id, month_end_11yr, month_start_14yr):
```

Comment:
can these methods be renamed. they are quite confusing because they dont get school information for everyone, just AG. it would be a lot clearer if that were added to the name